samples/stored-procedures/count.js

```diff
@@ -1,66 +1,66 @@
-﻿/**

```

Comment:
Is this file getting modified? Looks like it's not. Can we remove if it from the change?